rviz_default_plugins/src/rviz_default_plugins/displays/grid_cells/grid_cells_display.cpp

```diff
@@ -101,6 +101,12 @@ void GridCellsDisplay::updateAlpha()
   context_->queueRender();
 }
 
+void GridCellsDisplay::updateColor()
+{
+  cloud_->setColor(rviz_common::properties::qtToOgre(color_property_->getColor()));
```

Comment:
Since this change I'm getting a compile error. Is there another repo I need to pull?
```
/home/developer/workspaces/ros2-dev0/src/ros2/rviz/rviz_default_plugins/src/rviz_default_plugins/displays/grid_cells/grid_cells_display.cpp: In member function ‘void rviz_default_plugins::displays::GridCellsDisplay::updateColor()’:
/home/developer/workspaces/ros2-dev0/src/ros2/rviz/rviz_default_plugins/src/rviz_default_plugins/displays/grid_cells/grid_cells_display.cpp:106:11: error: ‘using element_type = class rviz_rendering::PointCloud {aka class rviz_rendering::PointCloud}’ has no member named ‘setColor’; did you mean ‘setPickColor’?
   cloud_->setColor(rviz_common::properties::qtToOgre(color_property_->getColor()));
           ^~~~~~~~
           setPickColor
```

Reply:
What do you mean? It's passing on CI AFAIK.

In general, I wouldn't expect anything to work without all the repositories in the `ros2.repos` file being up-to-date.

Reply:
The function was introduced in `rviz_rendering::PointCloud` in `rviz_rendering` with this PR. So if you recompile this repository, it should be fine.

Reply:
The issue persisted after deleting `build/rviz*` and `install/rviz*` and building with `--packages-above rcl`. However, it is gone after deleting the entire `build` and `install` folders and rebuilding everything.

I don't know what happened. It's almost like there is a missing `build_depend`, but I do see it in the package.xml https://github.com/ros2/rviz/blob/993d85666886ed3a4f9ac7cf214430dd25c2362d/rviz_default_plugins/package.xml#L35